### PR TITLE
Solana FFI improvements

### PR DESCRIPTION
**Pull request by rahulghangas**
Adds phash and nhash to `solana-ffi` mint function

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,7 @@ $(DEPS): .install-solana-ffi  ;
 	@touch $@
 
 clean:
+	rm -rf ./rust/target/release/build/solana-ffi-*
```
Comment (roynalnaruto) [inline Makefile]: We don't need this. This line is already present below.

**Review by roynalnaruto — APPROVED** [review]
Looks good, just a redundant change that you can remove.

File: Makefile (hunk 1 above)
Comment (rahulghangas) [reply Makefile]: Ok, I'll remove it

Comment (rahulghangas) [inline Makefile]:
```suggestion
```

**Comment by rahulghangas** [comment]
@roynalnaruto can you merge this please

**Pull request merged**